### ISSUE TITLE
Dynamicallly setting [accept]

### ISSUE THREAD

**Issue by code-wizard** [AckerApple/angular-file#13]
I am unable to bind the accept attribute ngfDrop. Here is my code

<div ngfDrop multiple="1"
             selectable="1"
             [accept]="accept"
             [ngClass]="{'another-file-over-class': validComboDrag}"
             [(validDrag)]="validComboDrag"
             [(files)]="files"
             class="well my-drop-zone"
        >

If the value of "accept" changes in the component, the directive is not updated.

**Comment by AckerApple** [comment]
Your issue report and claim, have been verified.

I forked this package and that code is left over from the original package.

Some notes for us on how to fix:
1. Currently, the accept bing is only applied in the [ngOnInit as seen here](https://github.com/AckerApple/angular-file/blob/development/src/file-upload/ngf.directive.ts#L52)
2. The dragover host listener does a [isFilesValid check as seen here](https://github.com/AckerApple/angular-file/blob/development/src/file-upload/ngfDrop.directive.ts#L44)
3. Upon drop, the ngf directive converts an event of files into an array of valid files using the Uploader class's [getValidFiles method as seen here](https://github.com/AckerApple/angular-file/blob/development/src/file-upload/ngf.directive.ts#L111)

**Quick History** leading to this issue: The original package I forked from had some Uploader middle class that was way over bloated doing way too much work including processing the accept attribute. Since a separate Uploader class was used, it has no binding to reread an updated accept attribute.

 **Suggested Fix**: Move file validation FROM FileUploader class into the base ngf directive. I've already separated and almost completed deprecated the FileUploader class we don't need but the validation of accept attribute still takes place there. Move validation and read binded this.accept at every validation step.

This is not a high demand request and will not be worked on immediately but certainly thinking this week certainly because that FileUploader class I despise is again causing static.

Will report when I have more
